bin/helpers.rb

```diff
@@ -0,0 +1,64 @@
+def download_file(file_name, remote_url)
```

Comment:
what do you think about breaking this file up into something like `file_system_helpers.rb` and `remote_repo_helpers.rb` ?

Reply:
I don't feel strongly against it but it feels like overkill to re-organize helpers for helpers, I was just thinking of a bucket to not repeat copy helpers that we are using in one place now but the trend is upwards. Can modify it if we think is very necessary

Reply:
Certainly not necessary.